lib/model.js

```diff
@@ -2521,7 +2521,7 @@ class Model {
    * @param  {Transaction}  [options.transaction]            Transaction to run query under
    * @param  {Function}     [options.logging=false]          A function that gets executed while running the query to log the sql.
    * @param  {boolean}      [options.benchmark=false]        Pass query execution time in milliseconds as second argument to logging function (options.logging).
-   * @param  {boolean}      [options.returning=false]        Append RETURNING * to get back auto generated values (Postgres only)
+   * @param  {boolean|Array} [options.returning=false]       If true, append RETURNING * to get back all values; if an array of column names, append RETURNING <columns> to get back specific columns (Postgres only)
```

Comment:
This array should be mapped to their field names. This will allow users to pass attributes like `createdAt` and we will query correct column `created_at` etc. Something like this https://github.com/sequelize/sequelize/blob/1cffab70382c5367fbb6ff611d11056f6410867c/lib/model.js#L2646

Reply:
@mikesadowski12 Please address sushantdhiman's comment as well :+1:

Reply:
just addressed it, although i am not entirely sure i did what he meant. let me know if i need to do anything else.

Reply:
@mikesadowski12 You did right :+1:

Just add a test to ensure this is followed, i.e., a test in which you pass `createdAt` and the generated query contains `created_at` (and not `createdAt`, which is what it would have if you didn't make this last change) (of course, you need the model to have `field: "created_at"` or at least `underscored: true`)

Let me know if you have any doubts

Reply:
PR looks good to go in current state

Reply:
should i add these tests then or is it ready to go with the removal of that file?

Reply:
@mikesadowski12 Ready to go, I believe :+1: